apps/site/assets/ts/schedule/components/SchedulePage.tsx

```diff
@@ -22,11 +23,22 @@ const SchedulePage = ({
     holidays,
     fare_link: fareLink,
     route_type: routeType,
-    schedule_note: scheduleNote
+    schedule_note: scheduleNote,
+    direction_names: directionNames,
+    direction_destinations: directionDestinations,
+    stops
   }
 }: Props): ReactElement<HTMLElement> => (
   <>
     {!!scheduleNote && <ScheduleNote scheduleNote={scheduleNote} />}
+    {routeType !== 0 &&
+    routeType !== 1 && ( // don't show for subway
```

Comment:
scheduleNote is only truthy for subway at the moment, so this could be a scheduleNote OR scheduleFinder instead. That could change so I guess this defensive logic is OK.